netsim/augment/links.py

```diff
@@ -1167,7 +1167,7 @@ def transform(link_list: typing.Optional[Box], defaults: Box, nodes: Box, pools:
       continue
 
     set_link_bridge_name(link,defaults)
-    link_default_pools = ['p2p','lan'] if link.type in ['p2p','lag'] else ['lan']
+    link_default_pools = ['p2p','lan'] if link.type in ['p2p','virtual_lag'] else ['lan']
```

Comment:
It might be better to implement #1647 than to tweak this list every time we have a new idea.